release_notes.md

```diff
@@ -0,0 +1,51 @@
+# Release Notes
+
+## Enhancements
+
+>**IMPORTANT**: This solution pack now requires FortiSOAR version `7.4.0` and later
+
+- Added the tag `CICD` to enhance searchability for the *Continuous Delivery* solution pack on Content Hub
+
+- Aligned the sequence of the task *Setup the Development Environment* with *Setup the Source Control for Production Environment*
+
+- Removed the dependency on the **Attachments** records to create export templates
+
+    - Added *Content and Settings Export Templates* in export template record
```

Comment:
i think extra space got added?

Reply:
It is for this effect

![image](https://user-images.githubusercontent.com/100339208/230374168-687b1c60-9eec-4b31-8563-c741afe682c4.png)